woocommerce/class-sv-wc-plugin.php

```diff
@@ -1454,6 +1460,17 @@ protected function set_dependencies( $dependencies = [] ) {
 		wc_deprecated_function( __METHOD__, '5.2.0' );
 	}
 
+	/**
+	 * Add the HPOS compatibility.
+	 *
+	 * @since x.y.z
+	 */
+	public function add_hpos_compatibility() {
```

Comment:
```suggestion
	public function handle_hpos_compatibility() {
```

it may not add it since it's conditional